src/docker-compose/dc.sh

```diff
@@ -6,19 +6,21 @@ fi
 SCDIR=$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")
 SCDIR=$(realpath $SCDIR)
 if [ "$1" == "" ]; then
-    echo "Usage is: "${BASH_SOURCE[0]}" <database> <broker> [options] [flags]"
+    echo "Usage is: "${BASH_SOURCE[0]}" <database> <broker> [options] [flags] [--no-wait]"
     echo "Where:"
     echo "    database: is one of mariadb, mysql or postgres"
     echo "    broker: is on of kafka or rabbitmq"
     echo "    options: one or more of ssl, zipkin, prometheus, influxdb, wavefront, ssl, dood, debug-dataflow, debug-skipper"
     echo "    flags: any docker-compose options"
+    echo "    --no-wait: Don't display command line and don't wait for input."
```

Comment:
Should we say what the default is?

Reply:
I decided that waiting for input on all executions is problematic and switched it around by adding a --wait option instead.